### DISCUSSION TITLE
So we're kinda fucked

### DISCUSSION THREAD

**Discussion by afkvido** [ProdigyPNP/ProdigyMathGameHacking#193] [Developer Forum]
# Hacks are going to be down for a while, and I can't fix them right now. The old discord also got terminated, so please join the temporary new one
# https://discord.gg/2qzTbdXSg9

**Comment by Poopykid768** [comment]
Whats the name of the new server

**Reply by afkvido** [reply]
ProdigyPNP Terminated or smth like that

**Reply by 123bov** [reply]
hi im new can i join to

**Reply by 123bov** [reply]
what is the discord code so i can join

**Reply by 123bov** [reply]
oh and how old are yall

**Reply by afkvido** [reply]
15

**Reply by S0vereignity** [reply]
I'm 14
Not tat anyone cares..

**Reply by HIHIHIHIHIHIHIHI-ui** [reply]
you spelled "that"wong

**Comment by helloeveryonehoware** [comment]
Fuck prodigy X for not helping

**Reply by ashtoncoded** [reply]
We already helped them and they betrayed us lol

**Reply by afkvido** [reply]
@AgrTheDev Unless this is a misinterpretation, you backdoored the obfuscated code.

**Reply by ashtoncoded** [reply]
Yeah because we knew you would betray us. Simple knowledge and proven right.

**Reply by ashtoncoded** [reply]
And I like how you can stalk the chat without responding. Bros lost for words

**Reply by afkvido** [reply]
I only betrayed y'all cuz i found malicious intent in the obfuscated code

**Reply by ashtoncoded** [reply]
We both know that’s a lie. 
1. You had no clue until everyone was 404ed
2. There was no way of you knowing what was in the obfuscated code.

**Reply by helloeveryonehoware** [reply]
Bro I'm not lost for words it's because I have a life unlike you @AgrTheDev

**Reply by ashtoncoded** [reply]
I don't even know you and who hurt you

**Reply by Nightlight8852** [reply]
What does prodigy x even do

**Reply by ashtoncoded** [reply]
It’s another menu

**Reply by helloeveryonehoware** [reply]
@afkvido you could downgrade it to 3.0.0 because right now it's 3.0.1 and if you downgrade it now, it potentially could undo the error code 404 or 48

**Reply by eris-webserv** [reply]
> @afkvido you could downgrade it to 3.0.0 because right now it's 3.0.1 and if you downgrade it now, it potentially could undo the error code 404 or 48

That wouldn't work. P-NP and Infinite Zero are broken, not the hacks.

**Reply by helloeveryonehoware** [reply]
alr

**Reply by afkvido** [reply]
> What does prodigy x even do

@Nightlight8852 [ProdigyX](https://github.com/ProdigyAPI/ProdigyX) is the only currently working prodigy hack. Use it if you need to hack Prodigy.

**Reply by eris-webserv** [reply]
> > What does prodigy x even do
> 
> @Nightlight8852 [ProdigyX](https://github.com/ProdigyAPI/ProdigyX) is the only currently working prodigy hack. Use it if you need to hack Prodigy.

:+1: (svido check your DMs)

**Reply by afkvido** [reply]
Alr

**Comment by Glitcherpicture** [comment]
Its ok we know how hard you put work into this literal game changing extension. so hope you fix soon

**Reply by afkvido** [reply]
Thanks!

**Comment by nahlgtm** [comment]
Shit I'm fucked

**Reply by 2241282** [reply]
Supt up sutpid

**Comment by nahlgtm** [comment]
I'ma get banned

**Reply by Nightlight8852** [reply]
I don't think there is a way to ban prodigy users completely, when he fixes the hacks which will take god knows how long, you can just start up a new account with the extension and BAMM level 100

This is again if you get banned

**Reply by nahlgtm** [reply]
Bro I had this for like 5 months

**Reply by HIHIHIHIHIHIHIHI-ui** [reply]
in old prodigy you can get banned

**Comment by 1234rj** [comment]
how do i hack

**Reply by afkvido** [reply]
Just use Prodigy X for now. ProdiygyPNP will be down for a hot while

**Reply by HIHIHIHIHIHIHIHI-ui** [reply]
what is prodigy X???
